src/routes/index.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const router = express.Router();
+
+/* GET home page. */
+router.get('/', (_, res) => {
+  res.render('index', {
+    title: 'Express',
+  });
+});
+
+/* POST home page */
+router.post('/', (req, res) => {
+  // Get the elements sent to the POST command from the post body
+  const elements = req.body['elements[]'];
+  let returnedString = '';
+
+  // Each element should be converted to proper HTML code
+  elements.forEach((element) => {
```

Comment:
Best Practices: This logic of this would be better represented as a controller that gets loaded into your route as this logic is no route related.